### PR TITLE
Implemented startplugin command

### PR DESCRIPTION
**Pull request by seocam**
startplugin it's a command that generates the base of a Colab plugin.
Just like django's startapp and startproject.

Signed-off-by: Sergio Oliveira <sergio@tracy.com.br>

### CODE REVIEW

**Comment by seocam** [comment]
@chaws thanks for the feedback.

You are right. A Colab plugin could be any kind of Django Plugin. If it's *any* I see some ways to approach:

1. Implement just simplest plugin possible
1. Implement two or three options and allow user to choose using parameters (`--type proxy` for example)
1. Implement just the most common case

I'd say that Django way would be to implement the simplest possible and document the others. Currently we don't have such a good documentation.

My idea here was to implement the most common case and afterwards move to options 2 or 3 as a second step.

How does it sound?

File: colab/conf/plugin_template/tests/plugins.d/app_name.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+menu_title = _('{{ app_name }}')
+
+url = colab_url_factory('{{ app_name }}')
+
+# Extra data to be exposed to plugin app config
+extra = {}
+
+menu_urls = (
+# Example of menu URL:
+#    url(display=_('Public Projects'), viewname='gitlab',
```
Comment (diegoamc) [inline colab/conf/plugin_template/tests/plugins.d/app_name.py]: `gitlab` is hardcoded here.

File: colab/conf/plugin_template/tests/plugins.d/app_name.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+#        kwargs={'path': 'public/projects'}, auth=False),
+
+# Example of authenticated user menu URL:
+#    url(display=_('Profile'), viewname='gitlab',
```
Comment (diegoamc) [inline colab/conf/plugin_template/tests/plugins.d/app_name.py]: `gitlab` is hardcoded here.

File: colab/conf/plugin_template/tests/test_plugin.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
+
+from django.test import TestCase, Client
+
+
+class PluginTest(TestCase):
```
Comment (diegoamc) [inline colab/conf/plugin_template/tests/test_plugin.py]: Shouldn't this class be named {{ app_name_camel }}PluginTest?

**Comment by MatheusFaria** [comment]
@seocam, Is this still needed? If so, rebase it.

**Comment by seocam** [comment]
@MatheusFaria yes, it is! Feel free to rebase! ;)

**Pull request merged**